playground-common/src/PSGenerator/Common.hs

```diff
@@ -187,13 +192,16 @@ someCardanoApiTxBridge = do
 
 miscBridge :: BridgePart
 miscBridge =
-    bultinByteStringBridge <|> byteStringBridge <|> integerBridge <|> scientificBridge <|> digestBridge <|> naturalBridge <|> satIntBridge <|> exBudgetBridge <|> someCardanoApiTxBridge
+    bultinByteStringBridge <|> byteStringBridge <|> integerBridge <|> scientificBridge <|> digestBridge <|> ratioBridge <|> naturalBridge <|> satIntBridge <|> exBudgetBridge <|> someCardanoApiTxBridge
 
 ------------------------------------------------------------
 
 psBigInteger :: PSType
 psBigInteger = TypeInfo "purescript-foreign-generic" "Data.BigInteger" "BigInteger" []
 
+psRatio :: PSType
+psRatio = TypeInfo "purescript-foreign-generic" "Data.Ratio" "Ratio" []
```

Comment:
Does the `FromJSON` instance of `Data.Ratio` on the PS side match the `ToJSON` instance of `PlutusTx.Ratio` ?